src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,131 @@
+package duke.command;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.Scanner;
+public class Duke {
+    private static final int TASK_MAX_SIZE = 100;
+    private static final int DONE_LENGTH = 5;
+    private static final int EVENT_LENGTH = 5;
+    private static final int TODO_LENGTH = 5;
+    private static final int DEADLINE_LENGTH = 8;
+
+    private static Scanner in = new Scanner(System.in);
+    private static int taskNumber = 0;
```

Comment:
I think this variable represents the number of tasks stored in the system. Maybe it is just me but I found the name "taskNumber" is a bit misleading (I thought it means the index of the current task). Maybe "tasksCount" is a better name?